lib/class-wp-theme-json-schema-v1-to-v2.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Class that implements a WP_Theme_JSON_Schema migrator.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that implements a migration from a v1 data structure into a v2 one.
+ */
+class WP_Theme_JSON_Schema_V1_To_V2 implements WP_Theme_JSON_Schema {
```

Comment:
I think one day we will need a v2 to v3 migration and v3 to v4...
Having a file "lib/class-wp-theme-json-schema-v1-to-v2.php" and class per migration  "WP_Theme_JSON_Schema_V1_To_V2" does not scale well. Files and classes are part of the public API.

Could we have a single migrate class that then has private methods inside per migration we need?